tests/NetEscapades.EnumGenerators.IntegrationTests/ExtensionTests.cs

```diff
@@ -14,8 +17,23 @@ public abstract class ExtensionTests<T> where T : struct
     protected void GeneratesToStringFastTest(T value)
     {
         var serialized = ToStringFast(value);
+        var valueAsString = value.ToString();
+        string? displayName = null;
 
-        serialized.Should().Be(value.ToString());
+        if (typeof(T).IsEnum)
+        {
+            if (valueAsString is not null)
+            {// Prevent: Warning CS8604  Possible null reference argument for parameter 'name' in 'MemberInfo[] Type.GetMember(string name)'
+                var memberInfo = value.GetType().GetMember(valueAsString);
```

Comment:
nit: not a blocker though!
```suggestion
                var memberInfo = typeof(value).GetMember(valueAsString);
```

Reply:
Actually `typeof(T)`. Fix it.